site/content/docs/main/csi-snapshot-data-movement.md

```diff
@@ -0,0 +1,401 @@
+---
+title: "CSI Snapshot Data Movement"
+layout: docs
+---
+
+CSI Snapshot Data Movement is built according to the [Volume Snapshot Data Movement design][1] and is specifically to move CSI snapshot data to a backup storage location.  
+CSI Snapshot Data Movement takes CSI snapshots through the CSI plugin in nearly the same way as [CSI snapshot backup][2]. However, it doesn't stop after a snapshot is taken. Instead, it tries to access the snapshot data through various data movers and back up the data to a backup storage connected to the data movers.  
+Consequently, the volume data is backed up to a pre-defined backup storage in a consistent manner.  
+After the backup completes, the CSI snapshot will be removed by Velero and the snapshot data space will be released on the storage side.  
+
+CSI Snapshot Data Movement is useful in below scenarios:
+- For on-premises users, the storage usually doesn't support durable snapshots, so it is impossible/less efficient/cost ineffective to keep volume snapshots by the storage, as required by the [CSI snapshot backup][2]. This feature helps to move the snapshot data to a storage with lower cost and larger scale for long time preservation.    
+- For public cloud users, this feature helps users to fulfil the multiple cloud strategy. It allows users to back up volume snapshots from one cloud provider and preserve or restore the data to another cloud provider. Then users will be free to flow their business data across cloud providers based on Velero backup and restore.  
+
+Besides, Velero [File System Backup][3] which could also back up the volume data to a pre-defined backup storage. CSI Snapshot Data Movement works together with [File System Backup][3] to satisefy different requirements for the above scenarios. And whenever available, CSI Snapshot Data Movement should be used in preference since the [File System Backup][3] reads data from the live PV and so is less consistent.  
```

Comment:
I wonder if it is worth explaining why reading off the live filesystem would be less desirable?

Reply:
Done, I added the general reason -- `since the File System Backup reads data from the live PV, in which way the data is not captured at the same point in time, so is less consistent.`.

I don't intend to elaborate it in details, as it will cover too many technical details.